src/components/Dropdown/Dropdown.css

```diff
@@ -0,0 +1,57 @@
+.Dropdown {
+  display: inline-block;
+}
+
+.Dropdown--button {
+  @apply text-pink no-underline flex w-full pl-20 ml-20;
+  .Icon {
+    @apply text-pink;
+  }
+  p {
+    @apply my-0 mx-12 leading-none text-sm;
+  }
+
+  .Dropdown--arrow {
+    @apply mt-4;
+  }
+}
+
+.Dropdown--list {
+  @apply list-reset my-8 py-8 shadow-md rounded-sm flex flex-col;
+  li {
+    @apply p-0;
+  }
+}
+
+.Dropdown--item {
+  @apply px-20 py-12 text-sm text-left border-l-2 border-white;
+  .Icon {
+    @apply text-grey mr-12;
+  }
+}
+
+.Dropdown--item:hover {
+  @apply text-pink border-pink;
+  .Icon {
+    @apply text-pink;
+  }
+}
+
+.Dropdown--container {
```

Comment:
I really like the css trick showing this menu with the `:active` selector, very clever 🤓 

We want to make sure that this menu is always centered under the button and that it is on top of any content below it. If it is not on top of the other content it will push the other content down the page when it's open which is an unexpected pattern for dropdowns.

To remedy this we want to position it absolutely to the top of it's parent then push it down the page using padding. What this does is it ensure the css content-box overlaps the the button so that when you hover the button it's actually hovering the container overlap. (green is padding overlap)
<img width="212" alt="Screen Shot 2019-04-19 at 4 53 01 PM" src="https://user-images.githubusercontent.com/1334131/56443828-44451180-62c4-11e9-9121-2ddea7934f71.png">

```css
.Dropdown--container {
@apply hidden w-full absolute;
  width: 200px;
  padding-top: 20px;
  margin-left: 25%;
  z-index: 100;
  top:0;
  left: -50%;

  &:hover{
    @apply block;
  }
}
```

This also uses a nifty centering trick with a negative left percentage that's then corrected by positive left margin